### PR TITLE
CIV-8574 Fix insufficient permission CUI

### PR DESCRIPTION
**Pull request by LeonardoPalmeiro**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CIV-8574

### Change description ###

remove the call to caseDocumentClientApi from CUI we're call just the docstore
image

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by laarmada — DISMISSED** [review]

**Review by kurt-uk — CHANGES_REQUESTED** [review]
We keeping the url to access the same?

**Review by kurt-uk — DISMISSED** [review]

**Review by hmcts-version1-ignacio — DISMISSED** [review]

File: src/integrationTest/java/uk/gov/hmcts/reform/civil/controllers/cases/DocumentControllerTest.java [hunk 1]
```diff
@@ -26,7 +26,6 @@
 import uk.gov.hmcts.reform.civil.model.docmosis.sealedclaim.Representative;
```
Comment (pats-john) [inline src/integrationTest/java/uk/gov/hmcts/reform/civil/controllers/cases/DocumentControllerTest.java:26]: Shouldnt there be a test at the Controller Layer to test these changes...

File: src/integrationTest/java/uk/gov/hmcts/reform/civil/controllers/cases/DocumentControllerTest.java [hunk 2]
```diff
@@ -26,7 +26,6 @@
...
 import uk.gov.hmcts.reform.civil.documentmanagement.model.CaseDocument;
```
Comment (pats-john) [inline src/integrationTest/java/uk/gov/hmcts/reform/civil/controllers/cases/DocumentControllerTest.java:27]: Wouldnt there be a Functional Test required for this Change...?

**Review by pats-john — CHANGES_REQUESTED** [review]

File: src/integrationTest/java/uk/gov/hmcts/reform/civil/controllers/cases/DocumentControllerTest.java (hunk 1 above)
Comment (LeonardoPalmeiro) [reply src/integrationTest/java/uk/gov/hmcts/reform/civil/controllers/cases/DocumentControllerTest.java]: you're right!! I refactored the documentControllerTest class as you suggest

File: src/integrationTest/java/uk/gov/hmcts/reform/civil/controllers/cases/DocumentControllerTest.java (hunk 2 above)
Comment (LeonardoPalmeiro) [reply src/integrationTest/java/uk/gov/hmcts/reform/civil/controllers/cases/DocumentControllerTest.java:10]: you're right!! I refactored the documentControllerTest class as you suggest

**Review by m-meulendijks-v1 — APPROVED** [review]

**Review by hmcts-version1-ignacio — DISMISSED** [review]

**Pull request closed without merging**